k8s/deploy/blueprint.yaml

```diff
@@ -0,0 +1,50 @@
+# The information below is boilerplate for the blueprint.
+apiVersion: xl/v1
+kind: Blueprint
+
+# This metadata helps the next user better understand what the blueprint is about.
+
+metadata:
+  projectName: Kubernetes YAML
+  description: |
+    The blueprint creates an application package with XL YAML files and Kubernetes YAML
+  author: XebiaLabs
+  version: 1.0
+
+parameters:
+
+- name: AppName
+  type: Input
+  description: What is the application name?
+- name: AppVersion
+  type: Input
+  description: What is the application version?
+  default: '1.0.0'
+- name: AppReplicas
+  type: Input
+  description: How many replicas of the application to deploy?
+  default: 1
+- name: K8sUsername
```

Comment:
Can you replace the `K8sUsername`, `K8sPassword`, `K8sToken` and `K8sApiServer` parameters with usage of the new function introduced by [LOVE-587](https://xebialabs.atlassian.net/browse/LOVE-587)? That will make the blueprint easier to use.

Reply:
Yes, I remember this feedback you gave during the integrations demo and it makes a lot of sense, reducing the number of inputs asked of the user. I'll replace these